util/start_test

```diff
@@ -697,7 +697,8 @@ def set_up_environment():
         #   libnuma which is only available as a .so)
         # - massivethreads (because it depends on various .sos)
         darwin = tgt_platform == 'darwin'
-        prgenv = compiler_utils.compiler_is_prgenv(chpl_compiler.get('target'))
+        origtargetcomp = chpl_compiler.get('target', llvm_mode='orig')
+        prgenv = compiler_utils.compiler_is_prgenv(origtargetcomp)
```

Comment:
Would using the chpl_cpu `target_is_prgenv` be an alternative here? If so I think it'd be worth moving that into compiler_utils (unless that introduces a bad cyclic dependency between `compiler_utils.py` and `chpl_compiler.py`